lookbook/docs/patterns/11-sub-header.md.erb

```diff
@@ -0,0 +1,49 @@
+This component is an own implementation for OpenProject requirements of contextual elements that affect the content of the page below. Such a component does not exist in the Primer Design System.
+
+## Anatomy
+
+**Filter input:** (Optional) It will change depending on the viewport:
+
+- **Desktop:** A basic [text input](https://primer.style/components/text-input) with a [search icon](https://primer.style/components/text-input#with-leading-and-trailing-visuals) which is mostly used on index pages to filter the elements below.
+- **Mobile:**
+  - There will be an IconButton shown instead of the input
+  - Clicking that, will show the input again and hide all other elements of the SubHeader
+  - An additional cancel button is shown to return to the previous state
+
+**Filter button:** (Optional)A button which is shown next to the filter input (or on the very left, if there is no filter input). Depending on what you need, it is either:
+
+- A normal button
+- A custom component, (e.g our core [FilterButtonComponent](../../inspect/open_project/filter/filter_button/default))
+
+**Text:** (Optional) A bold text shown in the middle of the SubHeader
+
+**Action buttons**: (Optional) A set of buttons that are shown on the right side
+
+
+## Best practices
+
+**Do**
+
+- Limit the actions of the SubHeader to the content specific actions of the page (e. for filtering, or creating a new object).
```

Comment:
@HDinger I would add to the page content to differentiate it from the actions in the PageHeader which are view/query related